packages/create-catalyst/src/utils/https.ts

```diff
@@ -118,10 +118,7 @@ export class Https {
     });
 
     if (response.status !== 200) {
-      console.error(
-        chalk.red(`\nPOST /device/token failed: ${response.status} ${response.statusText}\n`),
-      );
-      process.exit(1);
+      throw new Error('Device code not yet verified');
```

Comment:
🍹 If I ever ran into this error code, I would be confused why the command wasn't working. Could we elaborate here on why or suggest a path towards debugging it (even if it means submitting a bug report)?

P.S. this is a sign that we also need a debug command.

Reply:
This is a great point, ultimately this error is caught silently by the `poll` method in `login.ts`. It's basically a way to early return and retry. Maybe I should replace it with `return;`, since it's not technically an error? It's moreso just "The user hasn't logged into their store and entered their code yet".